demos/web-worker/.vscode/launch.json

```diff
@@ -11,7 +11,8 @@
       "url": "http://localhost:5002",
       "server": {
         "program": "${workspaceFolder}/node_modules/serve/bin/serve.js",
-        "args": ["-p", "5002"]
+        "args": ["-p", "5002"],
+        "logging": {"cdp": "${workspaceFolder}/vscode-pwa-cdp.log", "dap": "${workspaceFolder}/vscode-pwa-dap.log"}
```

Comment:
I like the idea of having these in the demos -- can you add those filenames to the root .gitignore so people don't accidentally commit them?

Reply:
Done